llvm/test/CodeGen/RISCV/pr84653_pr85190.ll

```diff
@@ -21,8 +21,7 @@ define i1 @pr84653(i32 %x) {
 ; CHECK-ZBB:       # %bb.0:
 ; CHECK-ZBB-NEXT:    sext.w a1, a0
 ; CHECK-ZBB-NEXT:    lui a2, 524288
```

Comment:
`lui` is unchanged here because it's a `X ^ ((1 << 31) - 1) -> X ^ ~(1 << 31)` case.